frontend/src/components/pages/CampOverview/MoveCamperModal/MoveModal.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import {
+  Button,
+  ModalBody,
+  ModalCloseButton,
+  ModalFooter,
+  ModalHeader,
+  Select,
+} from "@chakra-ui/react";
+import { Camper } from "../../../../types/CamperTypes";
+import { CampSession } from "../../../../types/CampsTypes";
+
+type PageTwoProps = {
+  camper: Camper;
+  campersToBeMoved: Camper[];
+  campSessions: CampSession[];
+  selectedCampSession: string;
+  setSelectedCampSession: (campSession: string) => void;
+  deselectAndClose: () => void;
+  moveCampers: () => void;
+};
+
+const MoveModal = ({
+  camper,
+  campersToBeMoved,
+  campSessions,
+  selectedCampSession,
+  setSelectedCampSession,
+  deselectAndClose,
+  moveCampers,
+}: PageTwoProps): JSX.Element => {
+  return (
+    <>
+      <ModalHeader>Move {campersToBeMoved.length} Camper(s)</ModalHeader>
+      <ModalBody>
+        <p style={{ marginBottom: "20px" }}>
```

Comment:
Can you change this to a `<Text>` or remove the p tag in some other way?

Reply:
Yep! Will do